### ISSUE TITLE
[Installing dbs] Error

### ISSUE THREAD

**Issue by damianosmel** [imgag/megSAP#92]
Dear megSAP team,

First, thank you for providing megSAP as open source code. 

I would like to install megSAP for the analysis of hearing loss genomic variants and their interpretation.

I do the installation on institute's server where I do NOT have sudo credentials. The admin run for me the followings after having installed the base dependencies:
```
cd megSAP/data
chmod 755 *.sh
sudo ./download_tools.sh
su
R -f install_deps_clincnv.R
```

Then, I continue without sudo credentials for the last two remaining installing commands:
```
./download_GRCh37.sh
./download_dbs.sh
```
The download and index of the genome works fine.
However, running download_dbs.sh, it complains:
"no BeSort in ngsbits=/home/<last_name>/megSAP/tools/ngs-bits/bin"
in the ClinGen group of commands:
$ngsbits/BedSort -in dosage_sensitive_disease_genes.bed -out dosage_sensitive_disease_genes.bed

Truly the contents of the ngs-bits/bin are:
```
GSvar_filters_cnv.ini  GSvar_filters.ini  GSvar.ini.example
libhts.a  libhts.so  libhts.so.1.7.0  libhts.so.2
settings.ini.example  settings_nightly.ini
```

During sudo ./download_tools.sh there is no error coming out by the ngs-bits installation.

I have cloned the release 0.2.

Thank you and looking forward to your feedback.

**Comment by marc-sturm** [comment]
Hi Damianos,

please use the master for installation.
The release 0.2 is very outdated (tools and databases) since it was released in 2019.
The master should work fine.

Best,
  Marc

**Comment by damianosmel** [comment]
Hi Marc,

thank you very much for the quick reply. Indeed after installing the correct qmake for centos (by the server admin), I got a bit further.

However, still the download_tools.sh gives errors on the centos server. The errors are coming from two points:
Installing python 3.6.9 environment: [bash block](https://github.com/imgag/megSAP/blob/c08f19a7169406b3aa78b850dc69b6445617cfd0/data/download_tools.sh#L149)

Installing REViewer: [bash block](https://github.com/imgag/megSAP/blob/c08f19a7169406b3aa78b850dc69b6445617cfd0/data/download_tools.sh#L173)

For the first one I have used my conda environment to install the needed dependencies (tensorflow for mmsplice) and then the splicing [tools](https://github.com/imgag/megSAP/blob/c08f19a7169406b3aa78b850dc69b6445617cfd0/data/download_tools.sh#L160) could be installed successfully.

During installation of the REViewer tool I get:
```

make[5]: *** [thirdparty/graph-tools/CMakeFiles/graphtools.dir/src/graphalign/GappedAligner.cpp.o] Error 1
make[4]: *** [thirdparty/graph-tools/CMakeFiles/graphtools.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [reviewer-prefix/src/reviewer-stamp/reviewer-build] Error 2
make[1]: *** [CMakeFiles/reviewer.dir/all] Error 2
make: *** [all] Error 2

```

Now my question is:
a) is the "hack" that I used my conda environment ok for later running megSAP? (Of course I will run it after load the respective conda env.)
b) can you suggest me a solution to solve the errors on installing REViewer tool.

For a) I would like to run megSAP to: vcf -> vep -> GSvar file, using /megSAP/src/NGS/an_vep.php.

Best,
Damianos

**Comment by marc-sturm** [comment]
Hi,

using conda environments or docker is fine for the splicing tools.

Unfortunately I cannot see the actual error message in the reviewer build error, so I cannot tell what the problem is.
Do you have a complete log of it?

Best,
  Marc

**Comment by damianosmel** [comment]
Hi Marc,

Thank you for re-opening.

After cleaning the REViewer folder and run again download_reviewer.sh (all lines from download_tools.sh about reviewer):
I get the output found at: download_reviewer_rm.out

The error comes from the not working downloading link for boost. I have found a similar [issue](https://github.com/Illumina/REViewer/issues/16) on reviewer tracker. To solve, I have update the download_reviewer.sh to select the master branch and not the one with the 0.1.1 tag. Please see this bash at: download_reviewer.sh

Indeed, this solves it and the boost is downloaded. However, still I get an error on installation of graphalign. Please see the output of installing reviewer at: download_reviewer_master.out

(all files are re-named as .txt in order to be able to upload them here)
[download_reviewer_rm.txt](https://github.com/imgag/megSAP/files/6629570/download_reviewer_rm.txt)

[download_reviewer.txt](https://github.com/imgag/megSAP/files/6629572/download_reviewer.txt)
[download_reviewer_master.txt](https://github.com/imgag/megSAP/files/6629573/download_reviewer_master.txt)

Any suggestion from your side to solve the graphalign installation error?

Thank you very much,
Damianos

**Comment by marc-sturm** [comment]
Hi Damianos,

I just tried building the master of reviewer and it worked.
I used gcc 7.5.0, cmake 3.10.2 and make 4.1 on Ubuntu 18.04.

I guess the problem is that your gcc 8.5 is not supported by reviewer.
You should open an issue in their tracker because of that.
Until the master of reviewer is fixed, you can try and compile it with gcc 7.5...

Best,
  Marc

**Comment by damianosmel** [comment]
Hi Marc,

I did update the gcc but still I was getting error on building the freebayes on the centos. To solve the issue, the server admin created a ubuntu docker for me and now the installation is complete.

For any other no-superuser that would like to install on centos, I would suggest to use ubuntu docker.

Best,
Damianos
